static/src/js/components/Banner/__tests__/Banner.test.js

```diff
@@ -27,7 +27,7 @@ describe('Bannner component', () => {
   test('should render self', () => {
     const { enzymeWrapper } = setup()
 
-    expect(enzymeWrapper.find('h2.banner__title').text()).toEqual('title')
+    expect(enzymeWrapper.find('h2.banner__title').text()).toEqual('<AlertHighPriority />title')
```

Comment:
I am a bit confused why the banner has HTML in it?

Reply:
shouldn't have that in there anymore